### ISSUE TITLE
New content idea: Ceph Drive Failure ML 

### ISSUE THREAD

**Issue by isabelizimm** [AICoE/content-pipeline#2]
## Topic
This is a python module created by the AIOps team that predicts disk health for Ceph.

## Background
As a data scientist, I want to establish the API for the disk health prediction python module, so that I can design the python module according to what kind of inputs to expect and what kind of output to return.

As a ceph developer, I want to establish the interface between the ceph manager daemon and the disk health prediction python module, so that I can ensure ceph manager is able to provide the inputs needed by this module in the correct format, and is able to use the output provided by it.

## Git artifacts, relevant resources
https://github.com/aicoe-aiops/ceph_drive_failure/issues/28

**Comment by MichaelClifford** [comment]
@chauhankaranraj does this  talk exist as its own issue? Can we close this?

**Comment by chauhankaranraj** [comment]
> @chauhankaranraj does this talk exist as its own issue? Can we close this?

Nope, there is no talk specifically about this python module. So yes, I think we can close this for now.
